templates/project.template.yaml

```diff
@@ -140,11 +71,11 @@ Resources:
               AWS:
                 - !If
                   - AddDevAccountPermissions
-                  - !Sub arn:aws:iam::${DevAwsAccountId}:role/SampleLambdaRole-${AWS::Region}-${DevAwsAccountId}-dev
+                  - !Sub ${DevAwsAccountId}
```

Comment:
Extending access to all roles and identities inside account to be able to use from future Lambda roles.